content/pages/disability-benefits/apply/supplemental-forms.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page-breadcrumbs.html
+title: Supplemental Disability Forms
+display_title: Supplemental Disability Forms
+description: Download additional forms you may need to apply for some VA disability benefits.
+concurrence: Incomplete
+plainlanguage: 
+template: detail-page
+order: 	
+relatedlinks:
+ - heading: 
+   links: 
+    - url: 
+      title: 
+      description:
+---
+
+<div class="va-introtext">
+
+Find out if you’ll need to turn in any supplemental forms when you file your disability claim. 
+
+</div>
+
+
+### Do I need to turn in any supplemental forms?
+
+In some cases, you may need to turn in one or more supplemental forms to support your disability claim. 
+
+<div class="va-table-overflow">
+<table class="va-table-explanatory">
+
+<thead>
+  <tr>
+    <th colspan="1" scope="col">If you’re:</th>
+    <th colspan="1" scope="col">You’ll need to turn in:</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+   <td><strong>A National Guard and Reserve member</strong></td>
+    <td>All your service treatment and personnel records from your unit(s)</td>
+  </tr>
+  <tr>
+   <td><strong>Claiming Individual Unemployability</strong></td>
+    <td><p><strong>Both of these:</strong></p>
+        <p>A Veteran’s Application for Increased Compensation Based on Unemployability (VA Form 21-8940)</p>
+        <p><a href="https://www.vba.va.gov/pubs/forms/VBA-21-8940-ARE.pdf">Download VA Form 21-8940</a>.</p>
+        <p><strong>and</strong></p>
+        <p>A Request for Employment Information in Connection with Claim for Disability Benefits (VA Form 21-4192)</p>
+        <p><a href="https://www.vba.va.gov/pubs/forms/VBA-21-4192-ARE.pdf">Download VA Form 21-4192</a>.</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Claiming Posttraumatic Stress Disorder (PTSD)</strong></td>
```

Comment:
Do we spell Posttraumatic/Post-Traumatic differently on the site than is spelled in the form titles?